src/test/kotlin/com/personio/synthetics/e2e/E2EBrowserTest.kt

```diff
@@ -23,6 +23,7 @@ class E2EBrowserTest {
             recoveryMessage("Test recovered")
             env("qa")
             tags("synthetics-api")
```

Comment:
Does it make sense to replace this one then?

Reply:
It would be nice to keep the correct version only then, even despite both versions are available, so that new repos would use the correct one. Wdyt?